tests/test_reports/test_trajectories_report.py

```
@@ -0,0 +1,517 @@
+_OK_STEP = _agent_step(0, msg="x", pt=5, ct=3)
+_OK_WIRE = _wire(0, 0, prompt=5, completion=3)
+_FAILED_WIRE = _wire(0, 0, prompt=5, completion=3, status_code=500)
+_ENRICHMENT_FIELD = {
+    "spliced": "trials_spliced_1_to_1",
+    "stashed": "trials_with_unmatched_calls_stashed",
+    "no_wire": "trials_with_no_wire_data",
+}
+
+
+@pytest.mark.parametrize(
+    "steps, wire_rows, no_steps, no_wire, enrich_kind",
```

Ruff (0.15.17)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)